src/api/app/controllers/webui/request_controller.rb

```diff
@@ -24,7 +24,7 @@ class Webui::RequestController < Webui::WebuiController
   before_action :check_ajax, only: :sourcediff
   before_action :prepare_request_data, only: %i[beta_show build_results rpm_lint changes mentioned_issues],
                                        if: -> { Flipper.enabled?(:request_show_redesign, User.session) }
-  before_action :cache_diff_data, only: %i[beta_show build_results rpm_lint changes mentioned_issues],
+  before_action :cache_diff_data, only: %i[changes],
```

Comment:
Where's the `request_diff_cacher` service the description talks about?

Reply:
Sorry the commit message has been updated. `request_diff_cacher` was removed